x-pack/legacy/plugins/xpack_main/server/lib/setup_xpack_main.js

```diff
@@ -22,9 +22,6 @@ export function setupXPackMain(server) {
 
   server.ext('onPreResponse', (request, h) => injectXPackInfoSignature(info, request, h));
```

Comment:
I suppose we can remove it as no plugins read it anymore. And remove tests https://github.com/elastic/kibana/tree/bf04235dae35452061cc7ea3d86d96c19a58206c/x-pack/test/licensing_plugin/legacy

Reply:
Ok, wasn't sure. Will do